source/appModules/foobar2000.py

```diff
@@ -71,7 +71,11 @@ def getElapsedAndTotalIfPlaying(self):
 			ui.message(_("No track playing"))
 		return elapsedAndTotalTime
 
-	def script_reportRemainingTime(self,gesture):
+	def script_reportRemainingTime(self, gesture):
+		import config
+		from languageHandler import setLanguage
+		lang = config.conf["general"]["language"]
+		setLanguage(lang)

```

Comment:
Are you sure you really need this code here? that script will be certainly run in NVDA's main thread, and core would have already called setLanguage.

Reply:
I've found the root problem of this, wx was overriding the locale